docs/scripting/trust_scripts/trust.me.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: trust.me
+---
+
+> trust.me
+
```

Comment:
```suggestion
```

If no description then no description?

Reply:
I feel like this one should be cryptic and left up to players to choose to trust or not so I deleted some fields. As far as the description field, I think I just did it that way so it appears normally on the autogenerated list page. I welcome conversation here about what should or should not be included.